db/schema.rb

```diff
@@ -75,6 +75,7 @@
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
     t.bigint "course_session_id"
+    t.date "schedule_date"
```

Comment:
Should this be just a date or should it be a date and time? With a time, you could also display the lecture time in the user's local timezone.

Reply:
I think definitely in general having time here is good practice. For the purposes of really just a fairly static date that the lectures typically are, we could go either way with this

Reply:
Yeah, I was thinking about a couple of different cases related to this, and thought it would be more likely that the Course Session would have a standard class time, so that all lectures within the session would start at the same time. For example, this session has always started at 3pm PT, but I could also potentially handle that as a default rather than omitting time.

Reply:
Agreed that this is the simplest implementation, I've just found that generally eventually people want full date/time and adding it in after the fact leads to incomplete data. However, this is not a blocker, just wanted to make sure you'd thought about it.

Reply:
I've gone through an updated the field to datetime. In doing so, I also set the config.time_zone parameter to be Pacific Time, so that when users set the time for 3pm in the form, that it doesn't use 1500 UTC, but the correct UTC time given the offset. Added a function to the model to handle formatting of the schedule_date in local time.